apps/examples/stdk_smart_switch/device_control.h

```diff
@@ -0,0 +1,99 @@
+/* ***************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <unistd.h>
+#include <tinyara/clock.h>
+#include <tinyara/config.h>
+
+#if defined(CONFIG_ARCH_BOARD_ARTIK05X_FAMILY)
+#define GPIO_OUTPUT_NOTIFICATION_LED 45
+#define GPIO_INPUT_BUTTON 44
+#define GPIO_OUTPUT_MAINLED 53
+
+#define GPIO_INPUT_BUTTON_EDGE IOTBUS_GPIO_EDGE_FALLING
+
+enum notification_led_gpio_state {
```

Comment:
The values for gpio signal,  iotbus_gpio_val_e are defined in iotbus/iotbus_gpio.h.
I think you'd better to use these values instead of 0, 1.

Reply:
fixed.